wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -906,6 +911,53 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:complexType>
 			</xs:element>
 			<!--===============================-->
+			<xs:element name="GetUserRoles">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="UserRole" type="tt:UserRole" minOccurs="0" maxOccurs="1">
```

Comment:
For GetUserRoles, why the UserRole full structure is passed in request, instead can't we just pass the UserRole Name parameter alone ?

Reply:
Good point. Included in the PR